services/QuillLMS/spec/models/app_setting_spec.rb

```diff
@@ -22,6 +22,25 @@
 RSpec.describe AppSetting, type: :model do
   let(:user) { create(:user) }
 
+  describe '#logged_out_enabled?' do
+    let!(:app_setting) do
+      create(:app_setting, name: 'egosum', enabled: true, percent_active: 100)
+    end
+
+    context 'AppSetting does not exist' do
+      it { expect(AppSetting.logged_out_enabled?(name: 'nemo')).to be false }
```

Comment:
If you use my above suggestion, you can use `let(:name) { 'nemo' }` with `subject` in this test.